review/management/commands/review_poller.py

```
@@ -187,13 +148,12 @@ def handle(self, *args, **opts):
-        self.token = settings.CASEWORK_POLLER_TOKEN
-        if not self.token:
-            raise PollerError(
-                "CASEWORK_POLLER_TOKEN is not set. Create a DRF token for the "
-                "poller's service account (manage.py drf_create_token <user>) "
-                "and set it in the environment."
+        try:
+            self.client = UpstreamClient(
+                on_log=self.stdout.write, on_err=self.stderr.write
             )
+        except UpstreamError as e:
+            raise PollerError(str(e))
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)